lib/cfa/grub2/install_device.rb

```diff
@@ -0,0 +1,93 @@
+require "cfa/base_model"
+
+module CFA
+  module Grub2
+    # specific parser for install devices.
+    # File format is easy element per line without comments.
+    # for better readability special values generic_mbr and activate is at
+    # the end of file
+    module InstallDeviceParser
+      # returns list of non-empty lines
+      def self.parse(string)
+        string.lines.map(&:strip).delete_if(&:empty?)
+      end
+
+      # gets list of devices and create file content from it
+      def self.serialize(data)
+        activate = data.delete("activate")
+        generic_mbr = data.delete("generic_mbr")
+
+        res = data.each_with_object("") do |disk, res|
```

Comment:
I find usage of `res` here to both the result and the block variable to be confusing.
What about?
```
res = data.each_with_object("") do |disk, disks|
```
Or `disks_string` or simply `string`... anything but `res`